_posts/2023-12-04-5.8.0-released.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: "Minetest 5.8.0 released!"
+author: rubenwardy
+description: >-
+  Minetest 5.8.0 has been released with improved onboarding, a new Settings
+  GUI, and improved Android controls
+image: /static/blog/5.8.0/cover.jpg
+image_credit: Wuzzy
+forum_topic: https://forum.minetest.net/viewtopic.php?t=30005
+tags:
+  - release
+
+mtg_debundle:
+  - src: /static/blog/5.8.0/debundle_no_games.png
+    caption: >-
+      What new players will see when they open up Minetest.
+
+  - src: /static/blog/5.8.0/debundle_cdb.png
+    caption: >-
+      Games you can install. This is the next thing we'll be redesigning.
+---
+
+<h2 class="sr-only">Introduction</h2>
+
+Minetest 5.8.0 has been released with improved onboarding, a new Settings GUI,
+improved Android controls, and a lot more.
+
+You can download Minetest from
+[our website](https://www.minetest.net/downloads/).
+Also, check out
+[the full changelog](https://dev.minetest.net/Changelog#5.7.0_.E2.86.92_5.8.0).
+
+<!-- more -->
+
+### Table of Contents
+
+- [Minetest no longer ships with a default game](#minetest-no-longer-ships-with-a-default-game)
+- [New Settings GUI](#new-settings-gui)
+- [Improved Android controls](#improved-android-controls)
+- [Rewritten sound support](#rewritten-sound-support)
+- [...and more](#and-more)
+- [Conclusion](#conclusion)
+
+
+## Minetest no longer ships with a default game
+
+Minetest is not a game - it's a game creation platform that allows you to make
+your own blocky gaming experiences. Developers can
+[make](https://rubenwardy.com/minetest_modding_book/) and share games. Players
+can [find](https://content.minetest.net/) and play games using the Minetest
+client.
+
+For a long time, Minetest shipped with a default game called
+[Minetest Game](https://content.minetest.net/packages/Minetest/minetest_game/).
+This game was intentionally left bare to encourage players to mod the game and
+try out other games for Minetest. However, this led to new users thinking that
+Minetest Game was all Minetest had to offer.
+
+From 5.8.0, users will instead be directed to choose a game when first starting
+Minetest. Minetest Game will still be available to download, but the user will
+see other games as well.
+
+We'll be improving the new user experience in the future, based on feedback. For
+example, we've already started a ContentDB dialog ("Browse online content")
+redesign to make it easier to find and choose packages.
+
+{% include figure_gallery.html items=page.mtg_debundle %}
+
+
+## New Settings GUI
+
+Minetest 5.8.0 comes with a new settings menu, designed to make it easier for
+new users to find and change settings.
+
+Don't worry about the appearance - we're going to be giving the entire main
+menu a makeover in a future release.
```

Comment:
Feels like this line could be improved. Perhaps something about it currently matching the current menu style, but the whole menu will be restyled in the future